examples/layers/raw/hello_world.dart

```diff
@@ -12,26 +12,37 @@ void beginFrame(Duration timeStamp) {
   final ui.Size logicalSize = ui.window.physicalSize / devicePixelRatio;
 
   final ui.ParagraphBuilder paragraphBuilder = ui.ParagraphBuilder(
-    ui.ParagraphStyle(textDirection: ui.TextDirection.ltr),
+    ui.ParagraphStyle(
+      textDirection: ui.TextDirection.ltr,
+      fontSize: 20,
+      fontWeight: ui.FontWeight.bold,
+    ),
   )
+    ..pushStyle(
+        ui.TextStyle(color: const ui.Color.fromARGB(0xFF, 0x33, 0x33, 0x66)))
```

Comment:
we should not change the color here. The default colors are supposed to be visible. See https://github.com/flutter/flutter/issues/76082